project/api/models/civi.py

```diff
@@ -131,7 +131,7 @@ def score(self, request_acct_id=None):
         D = -1
         A = 1
         SA = 2
-        owner_id = self.author
+        #owner_id = self.author
```

Comment:
Don't leave commented code. Either it is used, or it should be removed.

Reply:
ok owner_Id should be gone now.  Sorry about that.